testsuite/libra-fuzzer/README.md

```diff
@@ -20,10 +20,14 @@ To be effective, fuzzing requires a corpus of existing inputs. This
 crate contains support for generating corpuses with `proptest`. Generate
 a corpus with `cargo run generate <target>`.
 
-Once a corpus has been generated, the fuzzer is ready to use: run
-`cargo run fuzz <target>`. 
+Once a corpus has been generated, the fuzzer is ready to use, simply run:
 
-For more options, run `cargo run -- --help`.
+```
+RUSTC_BOOTSTRAP=1 cargo run fuzz <target> -- --release --debug-assertions
```

Comment:
I can't find where I read that `debug-assertions` was disabling `--release`, in any case it shouldn't be bad as long that it's not `--release` that disables `--debug-assertions` :D